hydroshare/hydroshare.py

```diff
@@ -713,6 +713,22 @@ def oracle():
         Discover.show_all(self.driver)
         oracle()
 
+    def test_B_000034(self):
+        """ Basic navigation to dashboard """
+
+        def oracle():
+            """ Expect get started to be showing """
+            self.assertTrue(Dashboard.is_get_started_showing(self.driver))
+
+        Home.login(self.driver, USERNAME, PASSWORD)
+        Dashboard.toggle_get_started(self.driver)
+        Dashboard.toggle_get_started(self.driver)
+        Home.to_home(self.driver)
+        oracle()
+        
+
 
 if __name__ == '__main__':
     parse_args_run_tests(HydroshareTestSuite)
+
+   
```

Comment:
Let's remove the extra lines at the end of the file here.

Reply:
good idea. Beta will have the dashboard in a  few days.

Reply:
beta.hydroshare.org

Reply:
changed to beta.